safe_transaction_service/events/services/queue_service.py

```diff
@@ -76,36 +76,48 @@ def get_queue_service():
 class QueueService:
     def __init__(self):
         self._connection_pool: List[BrokerConnection] = []
+        self._total_connections: int = 0
```

Comment:
No need for this variable, you can just use len of the connection pool, as it's quite fast: https://stackoverflow.com/a/1115329/724991

Reply:
Oh I get it! I misunderstood it